### DISCUSSION TITLE
Removing files from GIT  LFS

### DISCUSSION THREAD

**Discussion by cdennington** [community/community#28001] [Codespaces] [answered]
Referencing this https://github.com/orgs/community/discussions/22173 discussion

Are you able to delete LFS files for me on a private repo in the same way? Would I need to come to the discussions every time I want to do this or is there an easier way?

This is the repo I need to remove any files referenced by LFS: https://github.com/cdennington/paul-gawman

So far I've followed the steps here: https://gist.github.com/everttrollip/198ed9a09bba45d2663ccac99e662201 but Im still having issues when pushing to GitHub

**Comment by jshorty** [comment]
Hi there! Have you tried following the steps documented here?
https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage

I don't think this is related to Codespaces, so you may also want to consider recategorizing your question so that it gets broader visibility. Thanks!

**Comment by airtower-luna** [comment, marked as answer]
> Are you able to delete LFS files for me on a private repo in the same way? Would I need to come to the discussions every time I want to do this or is there an easier way?

Discussions isn't really the right place, you'd need to [contact support](https://support.github.com/request) as [described in the docs](https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage#git-lfs-objects-in-your-repository):

> If you need to purge a removed object and you are unable to delete the repository, please [contact support](https://docs.github.com/en/github/working-with-github-support) for help.

**Comment by cdennington** [comment]
@jshorty and @airtower-luna Thanks for your help

**Comment by ghost** [comment]


**Reply by airtower-luna** [reply]
That's for deleting a release, which is completely different from LFS.
